content/_data/changelogs/weekly.yml

```diff
@@ -7411,6 +7411,112 @@
+    category: regression
+    pull: 4701
+    issue: 62133
+    authors:
+    - daniel-beck
+    message: |-
+      When the resource root URL configuration pointed at a previously set up resource root URL, form validation showed a "404 Not Found" error. (regression in 2.205).
+  - type: major bug
+    category: regression
+    pull: 4696
+    issue: 62065
+    authors:
+    - romenrg
+    message: |-
+      Breadcrumbs displayed after notification alerts are again clickable.
+  - type: rfe
+    category: rfe
+    pull: 4685
+    issue: 61208
+    authors:
+    - timja
+    message: |-
+      Allow system read to view more admin monitors.
+  - type: rfe
+    category: rfe
+    pull: 4655
+    issue: 61841
+    authors:
+    - dwnusbaum
+    message: |-
+      Limit the number of exceptions thrown by some operations such as recursive directory deletion.
+      Previously, in rare cases, exceptions thrown when failing to delete large directories could consume significant amounts of memory.
+  - type: rfe
+    category: rfe
+    pull: 4668
+    authors:
+    - daniel-beck
+    message: |-
+      Indicate which component provides an URL that is always available without authentication in the global security configuration.
+  - type: bug
+    category: bug
+    pull: 4681
+    issue: 62006
+    authors:
+    - Vlatombe
+    message: |-
+      Fix a classloading issue while executing <code>ProcessTree.get()</code>.
+  - type: bug
+    category: bug
+    pull: 4687
+    issue: 62056
+    authors:
+    - daniel-beck
+    message: |-
+      Developer: Make it possible to look up extension implementations from more than one specific extension point at a time.
+  - type: rfe
+    category: developer
+    pull: 4693
+    issue: 61326
+    authors:
+    - uhafner
+    message: |-
+      Developer: Add <code>nogrid</code> option to <code>layout.jelly</code> tag to allow suppressing the bootstrap 3 grid.
+      See <a href="https://github.com/jenkinsci/bootstrap4-api-plugin">bootstrap4-api-plugin</a> for details.
+  - type: rfe
+    category: developer
+    pull: 4660
+    authors:
+    - jvz
+    message: |-
+      Developer: Update javax.mail to jakarta.mail 1.6.5.
+  - type: rfe
+    category: internal
+    pull: 4626
+    issue: 60866
+    authors:
+    - Wadeck
+    message: |-
+      Internal: Remove inline resources from ReverseProxySetupMonitor view.
+      Add a specific warning when the Jenkins Root URL does not contain the contextPath.
+  - type: rfe
+    category: internal
+    pull: 4627
+    issue: 60866
+    authors:
+    - Wadeck
+    - timja
+    message: |-
+      Internal: Remove inline resources from LogRecorder views.
+      Align the column headers of bigtables to the left
+
+  # pull: 4686 (PR title: Add form validation warning when signup is enabled)
+  # pull: 4691 (PR title: Maintainer guide: Reference the IRC channel)
+  # pull: 4692 (PR title: JENKINS-62088 Fix correct fonts not loading for buttons and plugin manager filter)
```

Comment:
Looks like legitimate bug, but up to @timja

Reply:
don't mind, it's a very minor bug, only had an issue in case it missed LTS cutoff and needed back porting